src/bitmat.c

```diff
@@ -488,7 +493,7 @@ static int _buffers (BITMAT *bm, int mode)
 /*--------------------------------------------------------------------*/
 
 int bm_allone (BITMAT *bm, int mode, int supp, int min, int max,
-               BMREPFN report, void *data)
+               BMREPFN report, void *data, int tacnt)
```

Comment:
/*Makh2018: Adding tacnt as parameter for bm_allone to delete full suppport items*/